src/components/styledComponents.js

```diff
@@ -0,0 +1,113 @@
+import styled from 'styled-components';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
import styled from "styled-components";
```